src/app/components/SubjectHeading/SubjectHeadings.jsx

```diff
@@ -0,0 +1,128 @@
+  }
+
+  componentDidMount() {
+    let {
+      fromLabel,
+      fromComparator,
+    } = this.props.location.query;
+    fromComparator = fromComparator.replace(/(^')|('$)/g, '');
+    fromLabel = fromLabel.replace(/(^')|('$)/g, '');
+    axios({
+      method: 'GET',
+      url: `${appConfig.shepApi}/subject_headings?from_label=${fromLabel}&from_comparator=${fromComparator}`,
+      crossDomain: true,
+      headers: {
+        'Access-Control-Allow-Origin': '*',
+        'Content-Type': 'application/json',
+      },
+    },
+    ).then(
+      (res) => {
+        this.setState({
+          previousUrl: res.data.previous_url,
+          nextUrl: res.data.next_url,
+          subjectHeadings: res.data.first_level_index,
+          error: res.data.first_level_index.length === 0,
+        });
+      },
+    ).catch(
+      (err) => {
+        console.log('error: ', err);
+        if (!this.state.subjectHeadings || this.state.subjectHeadings.length === 0) {
+          this.setState({ error: true });
+        }
+      },
+    );
+  }
+
+  extractParam(paramName, url) {
+    const params = url.replace(/[^\?]*\?/, '');
+    const matchdata = params.match(new RegExp(`(^|&)${paramName}=([^&]*)`));
+    return matchdata[2];
+  }
+
+  convertApiUrlToFrontendUrl(url) {
+    if (!url) return null;
+    const path = this.props.location.pathname;
+    return `${path}?fromLabel=${this.extractParam('from_label', url)}&fromComparator=${this.extractParam('from_comparator', url)}`;
+  }
+
+  redirectTo(url) {
+    const redirectFunction = function (e) {
+      e.preventDefault;
+      this.context.router.push(url);
+    }
+    return redirectFunction.bind(this);
+  }
+
+  pagination() {
+    const {
+      previousUrl,
+      nextUrl,
+    } = this.state;
+    const urlForPrevious = this.convertApiUrlToFrontendUrl(previousUrl);
+    const urlForNext = this.convertApiUrlToFrontendUrl(nextUrl);
+    return (
+      <div className="subjectHeadingNav">
+        <a className="subjectNavButton" href={urlForPrevious} onClick={this.redirectTo(urlForPrevious)}>{'\u25C0'}</a>
+        <a className="subjectNavButton" href={urlForNext} onClick={this.redirectTo(urlForNext)}>{'\u25B6'}</a>
+      </div>
+    )
+  }
+
+  render() {
+    const { error, subjectHeadings } = this.state;
+    if (error) {
+      return (
+        <div>
+            'No results found for that search'
+        </div>
+      )
+    }
+    return (
+      <div>
+        <div className="subjectHeadingsBanner">Subject Headings</div>
+        {this.pagination()}
+        <ul className="subjectHeadingRow">
+          <li>
+            <ul className="subjectHeadingLabelAndToggle">
+              <li className="subjectHeadingToggle"></li>
+              <li className="subjectHeadingLabel">Subject Heading</li>
+            </ul>
+          </li>
+          <li className="subjectHeadingAttribute">Titles</li>
+          <li className="subjectHeadingAttribute">Narrower</li>
+        </ul>
```

Comment:
Similar to above comment. I think we should use something beside `ul` and `li` here, since its one component, not a list of components.

Reply:
Also, eventually, we will want some styling to visual separate this from the list